core/embed/rust/src/ui/model_mercury/flow/danger.rs

```diff
@@ -22,13 +22,13 @@ use super::super::{
 };
 
 #[derive(Copy, Clone, PartialEq, Eq)]
-pub enum WarningHiPrio {
+pub enum Danger {
```

Comment:
I's slightly prefer naming this `ShowDanger` and also the file should be `show_danger.rs`. 
For consistency.